### ISSUE TITLE
Must : incorprate Session ID and USER ID

### ISSUE THREAD

**Issue by jitendrashahiitb** [advaitsaravade/Tree-Map-India#2]
The device will have  a log in for tree map app. ( see another issue)
this also presumes that the device is registered with server and has downloaded the updated list of users and their passwords (this is also a pending module to be developed)
The login asks if login as registered user or guest.
if already registered , click enables user to choose her name from a dropdown
if new , then question is: new registeration  as guest?
if new registration a form is filled
Form will have fields as described in the SRS and Procedure docs. Inter alia it includes, name , affiliation, Experience in surveying and allows a time to be set a 'breaking-time' where break-time is in minutes ( tentatively set as 10 Minutes) and is the amount of time that will be used to define , automatically a new session if there is no subsequent tree mapped within that time.

Thus login sets parameters like Name, area allocated for survey : in terms of city/prabhag/cluster etc to a commissined user i.e surveyor. Guest has freedom to choose what she likes.
The login gives a choice to either start survey or to upload data from device

All users must be either 1> registered or 2>guests
For registered user : login demands password.
Password can be reset if forgotton and if online. If not online, user will have to be as guest. the guest data will be committed only after a scrutiny.
Registerd user data will be committed with a validation process which may depend on REPUTATION of user, which is stored on server..
Session begins as soon as user logs in.

A session may be defined as continuos survey of trees. In normal cases, a tree takes about 2 minutes for formfilling and photographs,
It is therefore desirable that any break in survey of more than say 10 minutes be treated as different session.
Thus tha app should prompt the user the session number and should auto-increment the same after a break of say 10 minutes.
This time gap may be set by admin on the device either at online registration time of the device and be over ridden at the time of user login based on user profile.

the session ID is based on deviceid_surveyorid_date
